testapps/php56_custom_configs/nginx-custom.conf

```diff
@@ -76,11 +81,28 @@ http {
           fastcgi_read_timeout 600s;
         }
 
+        # This default behavior is intentionally using regex to allow
+        # users to override this with the exact match directive as follows:
+        # location = /_ah/health {
+        #   # Define your own behavior.
+        # }
+        #
+        location ~ ^/_ah/health$ {
+           access_log off;
+           error_log off;
+           return 200 'ok';
+        }
+
         # Allow users to specify their own configuration
         include /etc/nginx/conf.d/*.conf;
 
-	location /hello {
-          return 200 'hello from nginx-custom.conf';
+        # Drop accesses for some files
+        location = /favicon.ico { access_log off; log_not_found off; }
+        location ~ /\.          { access_log off; log_not_found off; deny all; }
+        location ~ ~$           { access_log off; log_not_found off; deny all; }
+
+        location /hello {
+            return 200 'hello from nginx-custom.conf';
```

Comment:
This is a config file for the test app. The test will access this path to make sure it is working.